alarm/alarm.py

```diff
@@ -0,0 +1,38 @@
+from gmail_handler import send_email
+
+EMAIL_TO_NOTIFY = "rjewing@bu.edu"  # Set to the email you want to notify
```

Comment:
Should we be pulling this from a db?

Reply:
or maybe another file (what if you want to notify multiple people idk)?

Reply:
Yeah you're right, we should allow multiple emails and pull from a config/database.